sunpy/data/_sample.py

```diff
@@ -3,10 +3,11 @@
 from __future__ import absolute_import
 
 from sunpy.util.net import url_exists
-from os import rename, remove
+from os import remove
 from astropy.utils.data import download_file
 from zipfile import ZipFile
 from urllib2 import URLError
+from shutil import move
 
 import os.path
 
```

Comment:
Not to do particularly with what's been changed in this PR, but following PEP:8ball: should the order of the imports be rearranged here? Like:
`__future__, os, shutil, zipfile, urllib2`  first
`astropy` then 
and `sunpy` aftewards?